presto-main/src/main/java/com/facebook/presto/operator/PagesHashStrategy.java

```diff
@@ -76,6 +76,13 @@
      */
     boolean positionEqualsRow(int leftBlockIndex, int leftPosition, int rightPosition, Page page, int[] rightChannels);
 
+    /**
+     * Compares the hashed columns in this PagesHashStrategy to the hashed columns in the Page. The
```

Comment:
nit: merge `The` to the next line.

Explain how does it different from `positionEqualsRow`. Currently they are using exactly the same wording.